include/riak_kv_ts.hrl

```diff
@@ -134,4 +134,10 @@
             'EXPLAIN' = ?SQL_SELECT{} :: ?SQL_SELECT{}
         }).
 
+-record(riak_sql_delete_query_v1,
```

Comment:
To avoid an (arguably unlikely) round of introducing `SQL_DELETE` macro for the current-version record when we happen to need a `riak_sql_delete_v2`, can it be done now, in this PR?